src/main/java/com/android/volley/AsyncCache.java

```diff
@@ -24,20 +24,49 @@
      */
     public abstract void get(String key, OnGetCompleteCallback callback);
 
-    public interface OnPutCompleteCallback {
-        /** Invoked when the put to the cache is complete. */
-        void onPutComplete();
+    public interface OnCompleteCallback {
```

Comment:
Maybe we should call this "OnWriteCompleteCallback" to distinguish it from OnGetCompleteCallback. The unifying factor in all of the places which use this callback is that it's an asynchronous write to the cache (where we don't care about the result of the write, just that it's done).

Reply:
Yeah, I think that'd be a good change.